string name
int32 state
int64 setpoint
bool enable
